run.py

```diff
@@ -1,10 +1,10 @@
 """
 This is the interactive Office Space Allocation command line interface
 Usage:
-	dojo create_room <room_type> <room_name>
-	dojo add_person <person_name> <FELLOW|STAFF> [wants_accomodation]
-	dojo (-i | --interactive)
-	dojo (-h | --help | --version)
+	Dojo create_room <room_type> <room_names>
+	Dojo add_person <person_name> <role> [<wants_accomodation>]
+	Dojo (-i | --interactive)
+	Dojo (-h | --help | --version)
 Options:
 	-i, --interactive Interactive Mode
 	-h, --help Shows the available commands for dojo
```

Comment:
These two options (-i and -h) don't work. Do you plan on adding functionality for them?